### PR TITLE
Fixed tr-up concept. Added a note to the tr-up shell.

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Some changes to wording regarding how tr-up handles snapshots.

### CODE REVIEW

**Review by tbazant — APPROVED** [review]
lg

File: xml/reference-transactional-update-usage.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
       The &tr-up-shell; command opens a shell in the transactional-update
       environment. In the shell, you can enter almost any linux command to make
```
Comment (dariavladykina) [inline xml/reference-transactional-update-usage.xml:29]:
```suggestion
      environment. In the shell, you can enter almost any Linux command to make
```

File: xml/reference-transactional-update-usage.xml [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
       changes to the file system, for example, install multiple packages with
-      the <command>zypper</command>. Or, you can verify that the changes you
+      the <command>zypper</command>, or perform changes to files that are part of the read-only file system. You can also verify that the changes you
       previously made with the &tr-up; command are correct.
```
Comment (dariavladykina) [inline xml/reference-transactional-update-usage.xml:32]:
```suggestion
      <command>zypper</command> or perform changes to files that are part of
      the read-only file system. You can also verify that the changes you
      previously made with the &tr-up; command are correct.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, a couple of nits. Thanks!

File: xml/reference-transactional-update-usage.xml [hunk 3]
```diff
@@ -26,9 +26,10 @@
     <title>The &tr-up; shell</title>
     <para>
       The &tr-up-shell; command opens a shell in the transactional-update
-      environment. In the shell, you can enter almost any linux command to make
+      environment. In the shell, you can enter almost any Linux command to make
       changes to the file system, for example, install multiple packages with
-      the <command>zypper</command>. Or, you can verify that the changes you
+      the <command>zypper</command>, or perform changes to files that are part
```
Comment (lvicoun) [inline xml/reference-transactional-update-usage.xml:31]:
```suggestion
      the <command>zypper</command> or perform changes to files that are part
```

File: xml/reference-transactional-update-usage.xml [hunk 4]
```diff
@@ -26,9 +26,10 @@
     <title>The &tr-up; shell</title>
     <para>
       The &tr-up-shell; command opens a shell in the transactional-update
-      environment. In the shell, you can enter almost any linux command to make
+      environment. In the shell, you can enter almost any Linux command to make
       changes to the file system, for example, install multiple packages with
-      the <command>zypper</command>. Or, you can verify that the changes you
+      the <command>zypper</command> or perform changes to files that are part
+      of the read-only file system. You can also verify that the changes you
```
Comment (dariavladykina) [inline xml/reference-transactional-update-usage.xml:32]:
```suggestion
      the <command>zypper</command> command or perform changes to files that
      are part of the read-only file system. You can also verify that the changes you
```

**Pull request merged**